src/clients/authorization/Settings.tsx

```diff
@@ -166,6 +166,7 @@ export const AuthorizationSettings = ({ clientId }: { clientId: string }) => {
           name="authenticationSettings"
           save={() => handleSubmit(save)()}
           reset={() => reset(resource)}
+          isActive={true}
```

Comment:
For boolean props that are true the value can be omitted.
```suggestion
          isActive
```

Reply:
fixed